infra/{{app_name}}/app-config/env-config/feature_flags.tf

```diff
@@ -0,0 +1,10 @@
+locals {
+  feature_flag_defaults = {
+    FOO = false
+    BAR = false
```

Comment:
Should this be empty (or commented out) in the template?

Doesn't really hurt anything, but potentially confusing/worth a callout in docs that you'll see a `FF_FOO`/`FF_BAR` env var in your runtime environment unless you delete these.

Similar for the override in `infra/{{app_name}}/app-config/dev.tf`

Reply:
Hmm yeah I was thinking that it doesn't hurt anything but good point that it could be confusing. I'll comment